server/service/src/sync/translations/invoice.rs

```diff
@@ -260,7 +260,7 @@ impl SyncTranslation for InvoiceTranslation {
             .map(|store_row| store_row.id);
 
         let invoice_type = invoice_type(&data, &name).ok_or(anyhow::Error::msg(format!(
-            "Unsupported invoice type: {:?} for {:?} mode",
+            "Unsupported invoice type: {:?} for (mode: {:?})",
```

Comment:
Site tracked, got confused by this message...

Reply:
not sure about the brackets. I would either remove or have them around the `for` as well 🤷 

```suggestion
            "Unsupported invoice type: {:?} for mode: {:?}",
```
or 

```suggestion
            "Unsupported invoice type: {:?} (for mode: {:?})",
```